examples/ace-axi/sc_main.cpp

```diff
@@ -24,6 +24,7 @@
 using namespace sc_core;
 using namespace axi;
 using namespace axi::pe;
+using namespace sc_core;
```

Comment:
residue from old pull-request https://github.com/VP-Vibes/SystemC-Components/pull/4
might not be needed anymore?